src/utils/string.js

```diff
@@ -1,4 +1,7 @@
-/* @flow strict */
+/* @flow */
+import { Clipboard } from 'react-native';
 
 export const transformToEncodedURI = (string: string): string =>
   string.replace(/\.\d/g, (match, p1, p2, offset, str) => `%${match[1]}`);
+
+export const copyStringToClipboard = (string: string): void => Clipboard.setString(string);
```

Comment:
I am skeptical that abstracting away the use of `Clipboard` is useful in this case.

Reply:
I was viewing a PR to gain some intuition as to what I might have left out, another maintainer had suggested as much abstraction as possible. And the reason they gave was that there are multiple instances where `Clipboard` might be used, and if it's abstracted it would be easier to refactor in case the parent API changes. I do agree though, `Clipboard` may not need to be abstracted. It has only two instances of use. Not worth losing the type-checking over. I'll rollback to the previous commit.